sdk/eventhub/azure-eventhub/azure/eventhub/aio/_producer_client_async.py

```diff
@@ -391,6 +441,37 @@ async def get_partition_properties(self, partition_id: str) -> Dict[str, Any]:
             EventHubProducerClient, self
         )._get_partition_properties_async(partition_id)
 
+    async def get_partition_publishing_properties(self, partition_id: str) -> Dict[str, Any]:
+        """Get the information about the state of publishing for a partition as observed by
+        the `EventHubProducerClient`. This data can always be read, but will only be populated with
+        information relevant to the active features for the producer client.
+            - `enable_idempotent_publishing` (bool)
+            - `partition_id` (str)
+            - `last_published_sequence_number` (Optional[int])
+            - `producer_group_id` (Optional[int])
+            - `owner_level` (Optional[int]
```

Comment:
I think the list might need spaces around it to render correctly

Reply:
I'll run a local sphinx to and do the necessary changes to make it being rendered properly.

Reply:
you're right! spaces added, thanks,